### PR TITLE
Match parse

### PR DESCRIPTION
**Pull request by aydedeEge**
Adds match and team scraping capabilities

### CODE REVIEW

File: Scraper/nbaStats/beautifulSoupSelenium.py [hunk 1]
```diff
@@ -76,11 +76,87 @@ def get_pmatches(args=None):
...
+        if args[3] == '-db':
+            mp.push_matches_to_db(matches)
+        elif args[3] == '-print':
+            for match in matches:
+                print(match)
+    
+    except Exception as e:
+        raise e
+
+# Right now only getting matches from 2016 onwards.
```
Comment (AsherWright) [inline Scraper/nbaStats/beautifulSoupSelenium.py:108]: Is this still the case or do we get all of them now?

File: Scraper/nbaStats/beautifulSoupSelenium.py [hunk 2]
```diff
@@ -76,11 +76,87 @@ def get_pmatches(args=None):
     except Exception as e:
         raise e
 
+# Do initial check to see if date in correct format
+def get_matches(args):
```
Comment (floregol) [inline Scraper/nbaStats/beautifulSoupSelenium.py:80]: get_pmatches too similar

File: Scraper/nbaStats/matchpage.py [hunk 3]
```diff
@@ -0,0 +1,320 @@
+import os
```
Comment (floregol) [inline Scraper/nbaStats/matchpage.py:1]: 2 files PLEASE

File: Scraper/nbaStats/matchpage.py [hunk 4]
```diff
@@ -0,0 +1,320 @@
...
+                all_team_ids_dict[team_name] = team_id
+
+                if team_name not in all_date_team_ids:
+                    all_date_team_ids[team_name] = team_id
+            except Exception as e:
+                pass
+ 
+        return all_team_ids_dict
+
+    # Add start_date and end_date args
```
Comment (AsherWright) [inline Scraper/nbaStats/matchpage.py]: Can we get better/different comments here? I don't know what this means. Is this a "TODO"?

File: Scraper/nbaStats/beautifulSoupSelenium.py [hunk 5]
```diff
@@ -76,11 +76,87 @@ def get_pmatches(args=None):
...
+            season_type="Regular%20Season"
+        )
+        print("Finished gathering all teams")
+    except Exception as e:
+        raise e
+
+    try:
+        matches = mp.get_matches_all_teams(
+            date=args[2],
+            season_type="Regular%20Season",
```
Comment (AsherWright) [inline Scraper/nbaStats/beautifulSoupSelenium.py:96]: nitpick: can we have this string as a global constant? we use it a lot.

File: Scraper/nbaStats/matchpage.py (hunk 3 above)
Comment (floregol) [reply Scraper/nbaStats/matchpage.py:1]: 3 FILES

**Review by AsherWright — COMMENTED** [review]
I like it, esp. since it's tested. One request: can we have a single line comment for each method just describing what it does? That way even someone like @WPoole could understand it.

**Review by floregol — APPROVED** [review]

File: Scraper/nbaStats/matchpage.py (hunk 3 above)
Comment (AsherWright) [reply Scraper/nbaStats/matchpage.py:1]: lol ^^

**Pull request merged**